spotbugs/etc/messages_fr.xml

```diff
@@ -3501,4 +3501,5 @@ double value2 =  x / (double) y;
  <BugCode abbrev="QBA">Assignation booléenne douteuse</BugCode>
  <BugCode abbrev="VR">Problème de compatibilité entre les versions</BugCode>
  <BugCode abbrev="DP">Utiliser doPrivileged</BugCode>
+ <BugCode abbrev="JUA">Problems in JUnit Assertions</BugCode>
```

Comment:
Did you mean to put an english message in the french messages bundle?

Reply:
`Problème in JUnit Assertions` ?

Reply:
[Google translation](https://translate.google.com/#view=home&op=translate&sl=en&tl=fr&text=Problems%20in%20JUnit%20Assertions) and [Deepl](https://www.deepl.com/translator#en/fr/Problems%20in%20JUnit%20Assertions) gave different answers for this sentence, so I will remove this change in this branch. It should work even in `fr` users (SpotBugs uses the English sentence instead).